### PR TITLE
[Fix] Fix Scripts for Populating OAuth Scopes [#OSF-7332]

### PR DESCRIPTION
**Pull request by cslzchen**
@sloria 

Update scripts to populate `ApiOAuth2Scope`.

This is part of https://openscience.atlassian.net/browse/OSF-7332.

### CODE REVIEW

File: scripts/register_oauth_scopes.py [hunk 1]
```diff
@@ -27,43 +26,38 @@ def get_or_create(name, description, save=True):
...
+    Given a dictionary of scope definitions, {name: scope_namedtuple}, load the
+    resulting data into a database collection
     """
+
     scope_dict = oauth_scopes.public_scopes
 
-    # Clear the scope collection and populate w/ only public scopes,
-    # nothing references these objects other than CAS in name only.
-    ApiOAuth2Scope.remove()
+    if clear:
```
Comment (cslzchen) [inline scripts/register_oauth_scopes.py]: The previous script clears the collection every time the scripts is run, which is preserved here as an option.

File: scripts/register_oauth_scopes.py [hunk 2]
```diff
@@ -74,12 +68,10 @@ def do_populate():
 
 
 def main(dry=True):
-    init_app(set_backends=True, routes=False)  # Sets the storage backends on all models
 
-    with TokuTransaction():
-        # Set storage backends for this model
-        set_backend()
```
Comment (cslzchen) [inline scripts/register_oauth_scopes.py]: I removed `set_backend()`.

File: scripts/register_oauth_scopes.py [hunk 3]
```diff
@@ -1,24 +1,23 @@
...
+
+django.setup()
 
 from scripts import utils as script_utils
 
 from framework.auth import oauth_scopes
-from framework.mongo import set_up_storage, storage
-from framework.transactions.context import TokuTransaction
-
+from osf.models import ApiOAuth2Scope
```
Comment (cslzchen) [inline scripts/register_oauth_scopes.py]: Is there a difference between `from website.oauth.models import ApiOAuth2Scope` and `from osf.models import ApiOAuth2Scope`?

Comment (cwisecarver) [inline scripts/register_oauth_scopes.py]: Unless you're checking `isinstance`,`issubclass`, or `==` then no. They should have the same behavior for querying.

File: scripts/register_oauth_scopes.py [hunk 4]
```diff
@@ -27,43 +26,38 @@ def get_or_create(name, description, save=True):
...
+    :return: the scope object
     """
+
     if name != name.lower():
         raise ValueError('Scope names are case-sensitive, and should always be lower-case.')
 
     try:
-        scope_obj = ApiOAuth2Scope.find_one(Q('name', 'eq', name))
-    except NoResultsFound:
+        scope_obj = ApiOAuth2Scope.objects.get(name=name)
```
Comment (cwisecarver) [inline scripts/register_oauth_scopes.py]: @cslzchen https://docs.djangoproject.com/en/1.10/ref/models/querysets/#get-or-create

Comment (cwisecarver) [inline scripts/register_oauth_scopes.py]: nm, that might not be as effective as what you're already doing.

**Comment by cwisecarver** [comment]
lgtm

**Pull request merged**